policyengine_canada/parameters/gov/provinces/nb/benefits/nbcb/base.yaml

```
@@ -0,0 +1,13 @@
+description: New Brunswick Child Tax Benefit program base
```

Similar phrasing and punctuation for other descriptions please
```suggestion
description: New Brunswick provides this base amount for its Child Tax Benefit.
```